interfaces/datastructure/Image.h

```diff
@@ -107,6 +107,14 @@ class SOLARFRAMEWORK_API Image {
         ENCODING_PNG
     };
 
+    // Rotate degrees
```

Comment:
* nit: add `char` underlying type.
* Not fan of the name/values, but nice to add this to public API so that it's not redefined everywhere.
Maybe not to be done in this PR if you want it to be merged ASAP, but what about `Rotation` instead of `RotateQuantity` which sounds weird?
As for the values, they are more acceptable (since we cannot start identifier by a number), but we could take inspiration from OpenCV or even [this one](https://learn.microsoft.com/en-us/dotnet/api/system.windows.media.imaging.rotation?view=windowsdesktop-9.0) from MS. What about:
```cpp
// OpenCV style
enum class : char Rotation
{
  ROTATE_NONE,
  ROTATE_90_CW,
  ROTATE_90_CCW,
  ROTATE_180
};
```
Or
```cpp
// MS-Style
enum class : char Rotation
{
  ROTATE_0,
  ROTATE_90,
  ROTATE_180,
  ROTATE_270
};
```